cmd/soroban-rpc/internal/methods/get_transactions_test.go

```diff
@@ -179,7 +179,7 @@ func TestGetTransactions_CustomLimitAndCursor(t *testing.T) {
 	assert.Equal(t, toid.New(3, 1, 1).String(), response.Cursor)
 
 	// assert transactions result
-	assert.Equal(t, 3, len(response.Transactions))
+	assert.Len(t, 3, len(response.Transactions))
```

Comment:
You need to drop the len() call here I think.

Reply:
ohh shoot right, fast fixes ..